dev-support/docker/Dockerfile

```diff
@@ -42,32 +42,11 @@ ENV DEBCONF_TERSE true
 ###
 RUN apt -q update \
    && apt install -y software-properties-common apt-utils apt-transport-https ca-certificates \
-   && add-apt-repository -y ppa:deadsnakes/ppa \
-   && apt-get -q install -y --no-install-recommends \
-        bash-completion \
-        build-essential \
-        bzip2 \
-        wget \
-        curl \
-        docker.io \
-        g++ \
-        gcc \
-        git \
-        gnupg-agent \
-        rsync \
-        sudo \
-        vim \
-        locales \
-        wget \
-        time \
-        openjdk-8-jdk \
-        python3-setuptools \
-        python3-pip \
-        python3.6 \
-        python3.7 \
-        python3.8 \
-        virtualenv \
-        tox
+   && add-apt-repository -y ppa:deadsnakes/ppa
+
+RUN mkdir /package
+COPY pkglist /package/pkglist
+RUN apt-get -q install -y --no-install-recommends $(grep -v '^#' /package/pkglist | cat)
```

Comment:
Could this just copy local_env_setup.sh and execute it? 
I suppose the only thing that would let us remove here would be `RUN pip3 install grpcio-tools mypy-protobuf`, so maybe it's not worth it.

Reply:
I thought about copying and executing local_env_setup.sh script, but since that one contains MacOS validations, I figured it would look odd living inside an ubuntu Dockerfile.

Reply:
Ack this makes sense, thanks